### ISSUE TITLE
[Bug - UI]: Have to be changed the Logo colors contrast

### ISSUE THREAD

**Issue by Toseef-Ahmad** [piyushgarg-dev/piyushgargdev-nextjs#569]
### Describe the bug

In the dark mode top bar logo does not have the proper colour contrast according to the colour wheel rules.
![image](https://github.com/piyushgarg-dev/piyushgargdev-nextjs/assets/65014307/66549bcf-5081-48ea-a7a2-1edd8857f6dc)

**More**
This logo is not attractive for viewing.

### To Reproduce

1. Go to the 'Home page'
2. Click on 'Dark mode toggle'
3. Scroll down to 'The top of the page'

### Expected behavior

The logo should have to be according to the colour wheel rules. And there has to be better colour contrast.

### Screenshots/Videos

![image](https://github.com/piyushgarg-dev/piyushgargdev-nextjs/assets/65014307/ed1ddb92-e71a-4cf1-9737-4dfea9e0e7d1)

### Please checkmark the following checklist

- [X] I make sure that similar issue is not opened
